app/controllers/application_controller.rb

```diff
@@ -1,4 +1,20 @@
 class ApplicationController < ActionController::Base
   # Prepend protect_from_forgery to the before_action chain manually
   protect_from_forgery prepend: true
-end
\ No newline at end of file
+
+  # Call the configured parameters
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  # Allow selected fields to be updated during mass assignment
```

Comment:
```
# Call the configured parameters
# Allow selected fields to be updated during mass assignment
```
are unnecessary. If the code is self-documenting, there is no need to add comments as they just add noise.

Reply:
Removed po, thank you :man_dancing: